api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -310,6 +319,9 @@ message AccessListReviewMetadata {
 
   // RemovedMembers are the members that were removed as part of the review.
   repeated string RemovedMembers = 6 [(gogoproto.jsontag) = "removed_members,omitempty"];
+
+  // AccessListTitle is the title of the reviewed access list.
+  string AccessListTitle = 7 [(gogoproto.jsontag) = "access_list_title,omitempty"];
```

Comment:
Same comment as above - I don't think this should be included here, instead include AccessListMetadata where needed.